.travis.yml

```diff
@@ -15,9 +15,9 @@ before_script:
   - unzip /tmp/radiotools.zip
   - mv radiotools-master radiotools
   - export PYTHONPATH=$PWD/radiotools
-  - wget https://github.com/nu-radio/NuRadioReco/archive/master.zip -O /tmp/NuRadioReco.zip
```

Comment:
Reminder to switch this to master again. In a pull request right after merging all.

Reply:
shouldn't we use python 3.7?